docs/en/credential-issuance-l2plus.rst

```diff
@@ -0,0 +1,671 @@
+
+Error Management
+----------------
+
+The error management MUST follow the same rules as defined in the Section :ref:`credential-issuer-endpoint:Credential Issuance Endpoints`, regarding the formats and the related standard references.
+
+During the MRTD PoP Validation flow, when recoverable errors occur, the MRTD PoP Service MAY generate and return a fresh nonce to enable the User to retry attempts while maintaining session security and preventing replay attacks.
+
+In addition to the error codes already defined in the Section :ref:`credential-issuer-endpoint:Credential Issuance Endpoints`, at least the following error codes MUST be supported.
+
+MRTD PoP Response Errors
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _table_eID_MRTD_PoP_Response_Errors:
+.. list-table:: MRTD PoP Response Error Codes
+   :widths: 30 70
+   :header-rows: 1
+
+   * - **Error Code**
+     - **Description**
+   * - **temporarily_unavailable**
+     - Document validation service or CIE Registry service is temporarily unavailable.
+
+MRTD PoP Validation Response Errors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _table_eID_MRTD_PoP_Validation_Response_Errors:
+.. list-table:: MRTD PoP Validation Response Error Codes
+   :widths: 30 70
+   :header-rows: 1
+
+   * - **Error Code**
+     - **Description**
+   * - **invalid_client**
+     - Wallet Instance authentication failed.
+   * - **invalid_request**
+     - HTTP Validation Request or Validation JWT is invalid or malformed (due to a malformed structure, missing data, signature failure, request timeout, etc.).
+   * - **access_denied**
+     - User authentication or document validation failed.
+
+HTTP Status Code Mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Error responses MUST use appropriate HTTP status codes:
+
+	- **400 Bad Request**: For ``invalid_request`` errors.
+	- **401 Unauthorized**: For ``invalid_client`` errors.
+	- **403 Forbidden**: For ``access_denied`` errors.
+	- **503 Service Unavailable**: For ``temporarily_unavailable`` errors.
+
+Security Considerations
+-----------------------
+
+Secure Session Management
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``auth_session`` parameter serves as the primary correlation mechanism between authentication steps. Implementations MUST ensure this identifier has sufficient entropy (minimum 128 bits) and is cryptographically secure. The session identifier MUST be validated at each step to prevent session fixation attacks.
```

Comment:
```suggestion
The ``auth_session`` parameter serves as binding proof between authentication steps. It MUST be cryptographically secured and use a sufficient entropy (minimum 128 bits). The ``auth_session`` value MUST be validated to prevent session fixation attacks.
```